contracts/LpSugar.vy

```diff
@@ -323,6 +323,21 @@ def forSwaps(_limit: uint256, _offset: uint256) -> DynArray[SwapLp, MAX_POOLS]:
   to_skip: uint256 = _offset
   left: uint256 = _limit
 
+  if _offset == 0:
+    convertor: IPool = IPool(self.convertor)
+    type: int24 = -1
+    if convertor.stable():
+      type = 0
+    pools.append(SwapLp({
+      lp: self.convertor,
+      type: type,
+      token0: convertor.token0(),
+      token1: convertor.token1(),
+      factory: empty(address),
+      pool_fee: 0 
+    }))
+    left -= 1
```

Comment:
Hmm, I'm not sure we need this. Could you please review line #387, I believe all we need to do is make an exception for the convertor is `reserve0 == 0`.

The convertor is returned by the factories we already have. We skip it in the `_pools()` to not display it in the front-end (since it doesn't allow deposits/withdrawals).

Reply:
You're right. Implemented the changes.